### PR TITLE
docs(express): Add v7 docs

### PR DESCRIPTION
**Pull request by chargome**
includes a fix for https://github.com/getsentry/sentry-docs/issues/3881

fixes https://github.com/getsentry/sentry-docs/issues/11514

### CODE REVIEW

**Review by mydea — APPROVED** [review]
This is so sweet! Great work!

**Review by lizokm — COMMENTED** [review]
I've made a few wording suggestions. Also got a bit of feedback from Rahul - he asked 1. If it would be possible to pull in the latest version number instead of just saying "latest" in the side nav. And 2. he said that there's a notification banner we use in the product that may look nicer than the yellow one we're using. (It already exists so we wouldn't need to ask the creative team to make anything for us).

File: platform-includes/getting-started-node/javascript.express__v7.x.mdx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  // The ID of error events captured by the Sentry error middleware is attached to `res.sentry`.
+  // You can use this ID for debugging, or to correlate requests with errors in Sentry.
+  res.statusCode = 500;
+  res.end(res.sentry + "\n");
+});
+
+app.listen(3000);
+```
+
+You can verify the Sentry integration by creating a route that will throw an error:
```
[the remark this answers is not in the document]
Comment (chargome) [reply platform-includes/getting-started-node/javascript.express__v7.x.mdx:117]: I think I would rather leave this one as is, as basically any route will do the job!

**Comment by chargome** [comment]
@lizokm thanks! I can include the remaining feedback in a separate PR 👍

**Pull request merged**